go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/beyondstorage/go-service-cos/v2 v2.2.0
 	github.com/beyondstorage/go-service-fs/v3 v3.4.0
 	github.com/beyondstorage/go-service-ftp v0.1.0
+	github.com/beyondstorage/go-service-gdrive v0.0.0-20210813025406-1a9502b99c89
```

Comment:
How about release a `v0.1.0` for `go-service-gdrive`? We need to make a release before we adding them into docs.

Reply:
Yep, we will release v0.1.0 after go-service-gdrive passed integration tests.

Reply:
v0.1.0 has been released, let's update the version here.